projects/core/src/product/occ/product.service.ts

```diff
@@ -26,7 +26,7 @@ export class OccProductService {
   loadProduct(productCode: string): Observable<Product> {
     const params = new HttpParams({
       fromString:
-        'fields=DEFAULT,averageRating,images(FULL),classifications,numberOfReviews'
+        'fields=DEFAULT,averageRating,images(FULL),classifications,numberOfReviews, potentialPromotions'
```

Comment:
please update that in unit test

Reply:
Had nothing to do with this PR, removed this change